CHANGES.md

```diff
@@ -14,6 +14,7 @@
   * Add `setHost` parameter for Consul CatalogNamer to set `authority` metadata 
   * Add auth `token` parameter to Consul Namer & Dtab Store
   * Add `datacenter` parameter to Consul Dtab Store
+  * Add `useHealthCheck` parameter to Consul Namer #589
```

Comment:
This was not part of the 0.7.4 release, can you move this to the "current" section (0.x.x)